templates/thank-you.html

```diff
@@ -0,0 +1,54 @@
+{% extends "contact-us/base_contact-us.html" %}
+
+{% block title %}Thank you{% endblock %}
+{% block head_extra%}<meta name="robots" content="noindex" />{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1uNruzpoTYxhrCJDxB0q_FH8z8YFs8w_m02fckevjFrU/edit{% endblock meta_copydoc %}
+{% block canonical_url %}https://ubuntu.com/contact-us/form{% endblock %}
+
+{% block content %}
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Thanks for getting in touch</h1>
+      <h2 class="p-heading--three">Your submittion was successful.</h2>
```

Comment:
So you decided not to add a "go back to {referer}" line?

Reply:
I was trying to work out how to pass parameters as a redirect.

Reply:
``` python3
def marketo_submit():
    # ...
    referrer = flask.request.referrer
    if referrer:
        return flask.redirect(f"/thank-you?referrer={referrer}")
    else:
        return flask.redirect(f"/thank-you")

def thank_you():
```

Or of course you could avoid the redirect with something like:

``` python3
def submit():
    # ...
    return flask.render_template(
        "submit.html",
        referrer=flask.request.referrer
    )
```